### PR TITLE
NO-ISSUE: Add OpenShift sandboxed containers operator

### PR DESCRIPTION
**Pull request by jhernand**
This is how the UI looks like:

image

### CODE REVIEW

File: libs/ui-lib/lib/common/config/docs_links.ts [hunk 1]
```diff
@@ -98,3 +101,6 @@ export const getCiscoIntersightLink = (downloadIsoUrl: string) =>
 
 export const MTV_LINK =
   'https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/2.7/';
+
+export const OSC_LINK =
+  'https://www.redhat.com/en/blog/exploring-openshift-confidential-containers-solution';
```
Comment (pezhang) [inline libs/ui-lib/lib/common/config/docs_links.ts:106]: @jhernand  Maybe it's better to use link https://docs.redhat.com/en/documentation/openshift_sandboxed_containers which is the OSC products documentation page. Thanks.

File: libs/ui-lib/lib/common/config/docs_links.ts [hunk 2]
```diff
@@ -67,6 +67,9 @@ export const HOST_REQUIREMENTS_LINK = 'https://access.redhat.com/solutions/48856
 export const ODF_REQUIREMENTS_LINK =
   'https://docs.redhat.com/en/documentation/red_hat_openshift_data_foundation';
 
+export const OSC_REQUIREMENTS_LINK =
+  'https://www.redhat.com/en/blog/red-hat-openshift-confidential-containers-deployment-solution';
+
```
Comment (pezhang) [inline libs/ui-lib/lib/common/config/docs_links.ts:72]: @jhernand How about using the https://docs.redhat.com/en/documentation/openshift_sandboxed_containers/1.8/html/user_guide/deploying-osc-bare-metal#osc-resource-requirements_deploying-bare-metal link here? Thanks.

Comment (jhernand) [reply libs/ui-lib/lib/common/config/docs_links.ts:72]: Sure, this was just an example. I will do that change and rebase.

File: libs/ui-lib/lib/common/config/docs_links.ts (hunk 1 above)
Comment (jhernand) [reply libs/ui-lib/lib/common/config/docs_links.ts]: Done.

File: libs/ui-lib/lib/common/config/docs_links.ts (hunk 2 above)
Comment (jhernand) [reply libs/ui-lib/lib/common/config/docs_links.ts:75]: Done.

**Comment by pezhang** [comment]
Hello @jhernand , I have copied this PR to https://github.com/openshift-assisted/assisted-installer-ui/pull/2719.  This PR can be closed. Thanks for drafting it.

**Pull request closed without merging**